README.md

```diff
@@ -1,19 +1,67 @@
-# Mine.js
+# Mine.js 🗻⚒
 
-> nodeJS implementation of an [Open Mined](http://openmined.org) data mine 🗻⚒
+> nodeJS implementation of an [Open Mined](http://openmined.org) data mine
 
-## Installation
+<!-- TOC depthFrom:2 -->
+
+- [🏃‍ Usage](#🏃‍-usage)
+    - [📦 Installation](#📦-installation)
+    - [🏁 Start](#🏁-start)
+- [⚙️ (Missing) Features](#⚙️-missing-features)
+- [🐞 Known Issues](#🐞-known-issues)
+- [⚖️ License](#⚖️-license)
+
+<!-- /TOC -->
+
+## 🏃‍ Usage
+
+### 📦 Installation
+
+The code was developed using node **v8.1.2** and it is recommend to stick around this version.
```

Comment:
Yes it's enforced via package.json. I'm using some node8 features like `await` that'd have to be `bablified` otherwise.
The whole thing currently doesn't feel stable enough to add tests and without tests I'm not confident to support wider version ranges.